bin/yaml/libraries.yaml

```diff
@@ -228,6 +228,14 @@ libraries:
       targets:
         - 1.37.0
         - 1.38.1
+    nsimd:
+      type: github
+      repo: agenium-scale/nsimd
+      build_type: manual
+      check_file: README.md
+      target_prefix: v
+      targets:
+        - 1.0.0
```

Comment:
@partouf this would be looking for a git tag `v1.0.0`, right?

The only tag on the repo is [v1.0](https://github.com/agenium-scale/nsimd/releases/tag/v1.0)

Reply:
lib.installation.FetchFailure: Fetch failure for https://github.com/agenium-scale/nsimd/archive/v1.0.0.tar.gz: <Response [404]>

haha! :) I'll fix

Reply:
@apmorton The version of NSIMD we want to use is on the 'spmd' branch of the repository. Should we put the 'spmd' target instead of v1.0? Even if the installation of this library is done manually?

Reply:
if you don't want to also install `v1.0` from github the changes to libraries.yaml should be reverted entirely.

Reply:
We will in this case delete these modifications.